src/manage_nsfs/health.js

```diff
@@ -361,7 +361,7 @@ class NSFSHealth {
                 config_data.nsfs_account_config.new_buckets_path;
 
             if (type === TYPES.ACCOUNT) {
-                const config_file_path = this.config_fs.get_account_path_by_name(config_file.name);
+                const config_file_path = this.config_fs.get_account_path_by_name(config_file);
```

Comment:
If you can change the `config_file` to `config_file_name` it can help readability.

Reply:
sure, done